es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.area = width * width;
+    this.circumference = width * 4;
+  }
+
+  print() {
+    return 'The area is ' + this.area + '. ' + 'The circumference is ' + this.circumference + '. ' + 'The width is ' + this.width + '.';
```

Comment:
Also override the print method to "print the Shape's message" followed by the "circle's radius". So.. we are not overriding the whole thing but we need to use Shape's print method as well

This goes same for Circle